gnomad_qc/v3/resources/sample_qc.py

```diff
@@ -1,69 +1,112 @@
+        {release: TableResource(f"{get_sample_qc_root(release)}/sample_qc_{strat}.ht") for release in RELEASES})
+
 
+def _get_ancestry_pca_ht_path(
+    part: str,
+    version: str = CURRENT_RELEASE,
+    include_unreleasable_samples: bool = False,
+) -> str:
+    """
+    Helper function to get path to files related to ancestry PCA
 
-def _get_ancestry_pca_ht_path(part: str, include_unreleasable_samples: bool = False) -> str:
-    return '{}/gnomad_v3_pca_{}{}.{}'.format(
-        SAMPLE_QC_ROOT,
+    :param part: String indicating the type of PCA file to return (loadings, eigenvalues, or scores)
+    :param version: Version of sample QC path to return
+    :param include_unreleasable_samples: Whether the file includes PCA info for unreleasable samples
+    :return: Path to requested ancestry PCA file
+    """
+    return "{}/gnomad_v{}_pca_{}{}.{}".format(
+        get_sample_qc_root(version),
+        version,
         part,
-        '_with_unreleasable_samples' if include_unreleasable_samples else '',
-        'txt' if part == 'eigenvalues' else 'ht'
+        "_with_unreleasable_samples" if include_unreleasable_samples else "",
+        "txt" if part == "eigenvalues" else "ht",
     )
 
 
-def get_ancestry_pca_loadings(include_unreleasable_samples: bool = False) -> TableResource:
+def ancestry_pca_loadings(include_unreleasable_samples: bool = False
+) -> TableResource:
     """
     Gets the ancestry PCA loadings TableResource
 
     :param include_unreleasable_samples: Whether to get the PCA that included unreleasable in training
     :return: Ancestry PCA loadings
     """
-    return TableResource(_get_ancestry_pca_ht_path('loadings', include_unreleasable_samples))
+    return VersionedTableResource(
+        CURRENT_RELEASE,
+        {release: TableResource(_get_ancestry_pca_ht_path("loadings", release, include_unreleasable_samples)) for release in RELEASES}
+    )
 
 
-def get_ancestry_pca_scores(include_unreleasable_samples: bool = False) -> TableResource:
+def ancestry_pca_scores(include_unreleasable_samples: bool = False
+) -> VersionedTableResource:
     """
     Gets the ancestry PCA scores TableResource
 
+    :param version: Version of sample QC path to return
     :param include_unreleasable_samples: Whether to get the PCA that included unreleasable in training
     :return: Ancestry PCA scores
     """
-    return TableResource(_get_ancestry_pca_ht_path('scores', include_unreleasable_samples))
+    return VersionedTableResource(
+        CURRENT_RELEASE,
+        {release: TableResource(_get_ancestry_pca_ht_path("scores", release, include_unreleasable_samples)) for release in RELEASES}
+    )
 
 
-def get_ancestry_pca_eigenvalues_path(include_unreleasable_samples: bool = False) -> str:
+def ancestry_pca_eigenvalues_path(include_unreleasable_samples: bool = False
+) -> VersionedTableResource:
     """
-    Gets the ancestry PCA eigenvalues path.
+    Gets the path to file containing ancestry PCA eigenvalues
```

Comment:
Docstring isn't quite accurate... this function returns a resource instead of a path and has no `version` parameter.